microsoft/testsuites/acctests/sgxtest.py

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+import pytest
+import subprocess
+
+from typing import Any
+from pathlib import Path
+from typing import Any
+from assertpy import assert_that
+
+from lisa import (
+    CustomScript,
+    CustomScriptBuilder,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.operating_system import Posix
+from lisa.tools import Echo, Uname
+
+########### HELPER METHODS ###################
+
+@staticmethod
+def getOEInstalledPath():
+    return os.path.expanduser("~/openenclave-install/")
+
+@staticmethod
+def getSGXVersion():
+    if isOeInstalled():
+        openenclave_install_path = getOEInstalledPath()
+        print("!!!!!!!!!!!!!!!!!!!!!!!!!!STARTED")
+        print(openenclave_install_path)
+        stdout = node.execute([openenclave_install_path + "bin/oesgx"], shell=True)
+        oesgx = stdout
+        sgx_version = ""
+        if "CPU supports Software Guard Extensions:SGX1" in oesgx:
+            sgx_version = "sgx1"
+        elif "CPU supports Software Guard Extensions:SGX2" in oesgx:
+            sgx_version = "sgx2"
+            
+    return sgx_version
+
+@staticmethod
+def isOeInstalled():
+    if not os.path.expanduser('~/openenclave-install/bin/'):
+        log.info("Not running test because openenclave binaries did not get installed.")
+        return False
+    return True
+
+
+def runOESamples(SAMPLES):
+    openenclave_install_path = getOEInstalledPath()
+    samples_stdout = ""
+    for sample in SAMPLES:
+        if sys.platform == "linux":
+            command = (
+                ". {}/share/openenclave/openenclaverc; make build; make run".format(
+                    openenclave_install_path
+                ),
+            )
+        elif sys.platform == "win32":
+            command = "vcvars64.bat & set OpenEnclave_DIR={}/lib/openenclave/cmake & cmake -G Ninja -DNUGET_PACKAGE_PATH=C:/oe_prereqs & ninja".format(
+                openenclave_install_path
+            )
+
+        returncode, stdout = execute(
+            command,
+            shell=True,
+            cwd="{}share/openenclave/samples/{}".format(
```

Comment:
Use `f` string to simplify the line like `f"{openenclave_install_path}share/openenclave/samples/{sample}"`